plugins/catalog-backend/src/ingestion/LocationReaders.ts

```diff
@@ -31,7 +31,6 @@ import { BitbucketApiReaderProcessor } from './processors/BitbucketApiReaderProc
 import { EntityPolicyProcessor } from './processors/EntityPolicyProcessor';
 import { FileReaderProcessor } from './processors/FileReaderProcessor';
 import { GithubReaderProcessor } from './processors/GithubReaderProcessor';
-import { GitlabApiReaderProcessor } from './processors/GitlabApiReaderProcessor';
```

Comment:
Accidentally part of this?

Reply:
Moved the wrong one at first, forgot to update here 😅 

Intending to remove these in the PR instead of duplicating, just to make sure git is able to follow the changes